Plugins/MongoDataStore/server/graphql/schema/mutations.graphql

```diff
@@ -0,0 +1,3 @@
+type Mutation {
+  setMongoDataStoreConfiguration(configuration: MongoDataStoreConfigurationInput!): MongoDataStoreConfiguration!
```

Comment:
It's a little weird that you require a response on the mutation but not on the query?

Reply:
will update